package.json

```diff
@@ -55,36 +55,36 @@
     "docs:serve": "vitepress serve docs"
   },
   "peerDependencies": {
-    "lexical": "0.14.5",
-    "vue": "^3.2.0"
+    "lexical": "0.15.0",
+    "vue": "^3"
```

Comment:
Can we put this back to `^3.2.0`? I want it to have this minimum version.

I think I put 3.2.x as minimum to make `Teleport` available. I'm not sure